api/ipfs/issuer.ts

```diff
@@ -0,0 +1,24 @@
+import {VercelRequest, VercelResponse} from '@vercel/node'
+import {HttpClient, HttpXhrBackend} from '@angular/common/http'
+import {catchError, tap, timeout} from 'rxjs/operators'
+import {EMPTY} from 'rxjs'
+// @ts-ignore
+import {XMLHttpRequest} from 'xmlhttprequest'
+import {IpfsService} from '../../src/app/shared/services/ipfs/ipfs.service'
+import {IPFSIssuer} from '../../types/ipfs/issuer'
+
+const httpClient = new HttpClient(new HttpXhrBackend({build: () => new XMLHttpRequest()}))
+const ipfsService = new IpfsService(httpClient)
+
+export default async (request: VercelRequest, response: VercelResponse) => {
+  const {hash} = request.query
+
+  return ipfsService.get<IPFSIssuer>(hash as string).pipe(
+    tap(issuer => response.status(200).send(issuer)),
+    catchError(_err => {
+      response.status(200).send({})
```

Comment:
@githinho, in case of error, is this good behavior, or do you want 400? how do you like it?

Reply:
I would rather return 502 - https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/502
Do you know which errors could accrue from ipfs?

Reply:
Leave the empty response.